docker/Dockerfile.exp

```diff
@@ -14,6 +14,7 @@ COPY . /src/ESMValTool
 RUN pip install -e .[develop]
 RUN mamba install -y -c conda-forge r-base
 RUN esmvaltool install R
+RUN mamba install -y -c conda-forge julia
```

Comment:
This should already come from the `environment.yml` in line 8, no? Perhaps it would be more important to remove line 5?

Reply:
that's a good point! I totally missed the bit where apt installs julia, am testing right now with that bit out and the mamba julia install out :beer:

Reply:
right, afraid that didn't work - same stuff as with the `Rscript` toolbox (I had to repair it a month or so ago by explicitly installing it after the whole env gets created). I don't really understand how this container works, but I think all the meat is done from the outside and it's just a silly way to test the installation of esmvaltool using a pre-built but not sourced Core env. Anyways, this is what I get:
```
Step 11/13 : RUN esmvaltool install Julia
 ---> Running in b2619b11d181
ERROR conda.cli.main_run:execute(33): Subprocess for 'conda run ['/bin/bash', '-c', 'esmvaltool install Julia']' command failed.  (See above for error)
ERROR:esmvalcore._main:Program terminated abnormally, see stack trace below for more information:
Traceback (most recent call last):
  File "/opt/conda/envs/esmvaltool/lib/python3.9/site-packages/esmvalcore/_main.py", line 479, in run
    fire.Fire(ESMValTool())
  File "/opt/conda/envs/esmvaltool/lib/python3.9/site-packages/fire/core.py", line 141, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
  File "/opt/conda/envs/esmvaltool/lib/python3.9/site-packages/fire/core.py", line 466, in _Fire
    component, remaining_args = _CallAndUpdateTrace(
  File "/opt/conda/envs/esmvaltool/lib/python3.9/site-packages/fire/core.py", line 681, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
  File "/src/ESMValTool/esmvaltool/install/__init__.py", line 36, in Julia
    self._run("julia", script)
  File "/src/ESMValTool/esmvaltool/install/__init__.py", line 20, in _run
    subprocess.check_output(
  File "/opt/conda/envs/esmvaltool/lib/python3.9/subprocess.py", line 424, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
  File "/opt/conda/envs/esmvaltool/lib/python3.9/subprocess.py", line 505, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/opt/conda/envs/esmvaltool/lib/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/opt/conda/envs/esmvaltool/lib/python3.9/subprocess.py", line 1821, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'julia'

installing Julia packages, please wait...

The command 'conda run --name esmvaltool /bin/bash -c esmvaltool install Julia' returned a non-zero code: 1
```

Reply:
Can I see the full logs somewhere? I am particularly interested to understand if Julia was part of the environment.yml based install and what might have gone wrong there or after that.

Reply:
no, unfortunately I didn't save the build log, but I double-checked - yes, Julia 1.6.3 was installed in the env

Reply:
here, retrieved from stdout, full log
[dockerlogexp.txt](https://github.com/ESMValGroup/ESMValTool/files/7539210/dockerlogexp.txt)